lib/iris/__init__.py

```diff
@@ -197,7 +197,14 @@ def __repr__(self):
         return msg.format(self.cell_datetime_objects, self.netcdf_promote,
                           self.netcdf_no_unlimited, self.clip_latitudes)
 
+    deprecated_options = {}
```

Comment:
We need to reinstate this because the extest helper module `extest_util.py` relies on it. More pertinently, it probably should not have been completely removed when the strict grib load deprecated option was removed, as we are likely to need this again as soon as we deprecate all the current options at v2.0. So let's just reinstate it, empty, now, in readiness for that and to prevent having to change `extest_util.py` unnecessarily.